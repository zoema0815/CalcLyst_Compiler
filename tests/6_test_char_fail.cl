int main () {
    char c;
    c = "Hello";
    return 0;
}