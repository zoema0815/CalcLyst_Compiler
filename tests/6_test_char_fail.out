Fatal error: exception Failure("illegal assignment char = String in c = Hello")
